ui/src/app/edge/index/widget/consumption/consumption.component.html

```diff
@@ -0,0 +1,28 @@
+<ng-container *ngIf="service.currentEdge | async as edge">
+    <ng-container *ngIf="edge.currentData | async as currentData">
+        <ion-card>
+            <ion-item color="light">
+                <ion-img style="width:40px;" src="assets/img/consumption.png">
```

Comment:
Ich weiß, das CSS ist noch von mir. Können wir darauf mittlerweile verzichten? - z. B. mit ion-avatar (https://ionicframework.com/docs/api/avatar) oder ion-thumbnail (https://ionicframework.com/docs/api/thumbnail); vielleicht auch mit ion-img.

Reply:
Hier ist auch noch etwas dazu: https://golb.hplar.ch/2018/01/Custom-SVG-icons-in-Ionic.html#ionic-4